R/addins.R

```diff
@@ -1,8 +1,115 @@
-read_addin <- function(...) {
-  path <- fs::path_package("rhino", "rstudio", "addins", ...)
-  readChar(path, file.info(path)$size)
+run_background <- function(file) {
+  path <- fs::path_package("rhino", "rstudio", "addins", file)
+  rstudioapi::jobRunScript(path, workingDir = rstudioapi::getActiveProject())
 }
 
 addin_module <- function() {
```

Comment:
This module needs some tuning.
1. It is not clear to the user, where the file will be added.
2. Rhino app can have a more complex structure and a user might want to add a module in e.g. a subdirectory of `app/view` or a totally different directory.